transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_cloud_licenses.sql

```diff
@@ -0,0 +1,12 @@
+with stripe_licenses as (
```

Comment:
No need for the CTE. The statement should be enough.

Reply:
Removed CTE.